application/core/plugins/dateFunctions/EMFunctions.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * This file is part of dateFunctions plugin
+ * @version 1.0.0
+ */
+
+namespace dateFunctions;
+
+use LimeExpressionManager;
+use Survey;
+
+class EMFunctions
+{
+    /**
+     * Formats a date according to the Survey's date format for the specified language
+     * @param string $date : a date in "Y-m-d H:i:s" format. Example: VALIDFROM.
+     * @param string $language : the language used for localization. If the language is not configured in the survey, the base language will be used. Example: TOKEN:LANGUAGE
+     * @return string
+     */
+    public static function localize_date($date, $language)
```

Comment:
In my opinion : 

1. $language must be optional : get it for App()->getLanguage() if null
2. add an opt for time